samples/zpp/zpp.rst

```diff
@@ -0,0 +1,10 @@
+.. _zpp-samples:
+
+zpp C++ Samples
+###############
+
+.. toctree::
+   :maxdepth: 1
+   :glob:
+
+   **/*
```

Comment:
The doc build error says it all.  There are no .rst files in the samples/zpp/* samples folders.  You'll need to add at least one to make this warning go away.

Reply:
Yeah i didn't notice the other code had README.rst and not README.txt, will be reworking things anyway, I'll ping you directly if I am at a point where I need a real documentation review.